build.gradle

```diff
@@ -89,7 +89,7 @@ subprojects {
         depends = [
                 guava: 'com.google.guava:guava:18.0',
                 findbugs: 'com.google.code.findbugs:jsr305:1.3.9',
-                junit: 'junit:junit:4.6',
+                junit: 'junit:junit:4.12',
```

Comment:
Do all of their existing tests still compile, run and pass?

Reply:
Yup.. I checked :)